upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -223,6 +223,7 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 		return strings.Join(labels, ",")
 	}
 
+	dest["IsIPv6Only"] = tf.IsIPv6Only
```

Comment:
Would you be ok with something like this? It would make the template more clear.
At this stage, it is irrelevant if cluster is IPv4 or IPv6, but only if they should be enabled or not.
Later, if we decide to add DualStack, we can keep the template and just modify the logic in the function(s).
```suggestion
	dest["enableIPv4"] = !tf.IsIPv6Only
	dest["enableIPv6"] = tf.IsIPv6Only
```